apps/src/templates/instructions/ReviewTab.jsx

```diff
@@ -25,6 +34,23 @@ export default class ReviewTab extends Component {
           token: request.getResponseHeader('csrf-token')
         });
       });
+
+    codeReviewDataApi
+      .getPeerReviewStatus(channelId, serverLevelId, serverScriptId)
+      .done(data => {
+        const id = (data && data.id) || null;
+        this.setState({
```

Comment:
should we be setting `loadingReviewableState` before this api call (and resetting to false when we get a result)?

Reply:
So `loadingReviewableState` controls showing the spinner in place of the checkbox next to "Enable Peer Review", but before the API call returns, the entire checkbox and label is hidden. I can add a different sort of loading flag here though if that helps? I'll add some screencaps to the PR as well to show the UI flow

Reply:
it might be good to show an error if we fail to load, because otherwise they won't know something went wrong on our end, but it's probably not a high priority if it complicates things.

Reply:
Yep makes sense, I'll add something here/add a task to follow up with messaging for this error case.